.pre-commit-config.yaml

```diff
@@ -14,7 +14,7 @@ repos:
         exclude: (CHANGELOG|TWEET).md
 
   - repo: https://github.com/pre-commit/mirrors-prettier
-    rev: v3.1.0
+    rev: v4.0.0-alpha.8
```

Comment:
why does this auto-upgrade to alphas o.O

Reply:
yeah, there's an issue on their repo, but it's a wontfix :'D